spinnaker_camera_driver/CMakeLists.txt

```diff
@@ -101,19 +101,6 @@ install(TARGETS
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
-# Redistributing the flycapture .so file is permitted by the SDK EULA:
-# http://www.ptgrey.com/support/kb/data/PGR-FlyCap-SDK-LA.pdf
-if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/usr/lib)
-  install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/usr/lib/ DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
```

Comment:
Looks like we're not bundling the sdk anymore. Do we not need this for the deb?

Reply:
I'm pretty sure that license agreement is outdated (there is no flycapture / pointgrey anymore, it's Spinnaker / FLIR now) and I think it is better to let the users install the SDK by themselves. We could try to make a separate ROS package that just catkinizes the Spinnaker SDK but I won't have time for that. The official way of installing the SDK also includes an explicit manual confirmation of the license.

So my suggestion is to use the download script just for CI and encourage the user to use this package from source.
Any better ideas?